src/app/workspace/resource/representation/video/video.component.html

```diff
@@ -0,0 +1,81 @@
+<div class="player" appDisableContextMenu [class.cinema]="cinemaMode">
+    <div class="container">
+
+        <video class="video" #videoEle *ngIf="!fileHasChanged; else loading"  [src]="video" type="video/mp4" preload="auto" [muted]="muted"
+            (click)="togglePlay()" (timeupdate)="timeUpdate($event)" (loadedmetadata)="loadedMetadata($event)"
+            (canplaythrough)="loadedVideo()">
+        </video>
+
+        <ng-template #loading>
+            <app-progress-indicator></app-progress-indicator>
+        </ng-template>
+
+        <div class="preview-line">
+            <!-- preview image -->
+            <div class="preview" #preview>
+                <app-video-preview
+                    [src]="src"
+                    [time]="previewTime"
+                    [dispTime]="true"
+                    [fileHasChanged]="fileHasChanged"
+                    (loaded)="displayPreview(!$event)">
+                </app-video-preview>
+
+            </div>
+        </div>
+    </div>
+
+    <mat-toolbar class="controls">
+        <mat-toolbar-row style="overflow: hidden;">
+            <app-av-timeline #timeline [value]="currentTime" [min]="0" [max]="duration" [resized]="cinemaMode"
+                (move)="updatePreview($event)" (mouseenter)="displayPreview(true)" (dimension)="timelineDimension = $event"
+                (mouseleave)="displayPreview(false)" (changed)="updateTimeFromSlider($event)"></app-av-timeline>
+        </mat-toolbar-row>
+
+        <mat-toolbar-row class="action">
+            <button mat-icon-button (click)="goToStart()" [disabled]="currentTime === 0"
+                matTooltip="Stop and go to start" [matTooltipPosition]="matTooltipPos">
+                <mat-icon>skip_previous</mat-icon>
+            </button>
+            <button mat-icon-button (click)="togglePlay()" [matTooltip]="(reachedTheEnd ? 'Replay' : (play ? 'Pause' : 'Play'))"
+                [matTooltipPosition]="matTooltipPos">
+                <mat-icon>{{ reachedTheEnd ? "replay" : (play ? "pause" : "play_arrow") }}</mat-icon>
+            </button>
+            <span class="empty-space"></span>
+            <span class="fill-remaining-space"></span>
+            <button mat-icon-button (click)="updateTimeFromButton(-10)"
+                matTooltip="10 seconds backward" [matTooltipPosition]="matTooltipPos">
+                <mat-icon>replay_10</mat-icon>
+            </button>
+
+            <div (wheel)="updateTimeFromScroll($event)">
```

Comment:
<img width="246" alt="Screen Shot 2022-04-21 at 14 18 43" src="https://user-images.githubusercontent.com/60604010/164456925-a9952152-f8b2-4f82-bfd8-0aac18e78591.png">
It's not a big thing and doesn't require a change in my opinion if it's not an easy fix but if I am replacing a video with a shorter video, then the progress from the previous video remains. If possible, this time should be reset to 00:00

Reply:
resolved in e4e740bc